### DISCUSSION TITLE
Question: Created a port of the first week of the game on Scratch any errors?

### DISCUSSION THREAD

**Discussion by Shmish21** [FunkinCrew/Funkin#1566] [Q&A]
I made a FNF Port of Week 1 on [Scratch](https://scratch.mit.edu/) it is supposed to be as close to the original as possible with no help from source code, just trying to code from gameplay. I just wanted to know if there are any such errors in the remake or if there was anything I am missing that needed to be added. [Here is the project link](https://scratch.mit.edu/projects/535973006/).

**Comment by ghost** [comment]
Looks good
